R/plotScoreHeatmap.R

```diff
@@ -199,9 +259,45 @@ plotScoreHeatmap <- function(results, cells.use = NULL, labels.use = NULL,
         args$color <- color
     }
 
+    # Troubleshooting and testing purposes
+    if (!is.null(args$return.data) && args$return.data) {
+        return(args)
+    }
```

Comment:
I wonder your thoughts on this bit @LTLA. It'd give me a way to write unit tests for many of the things that currently require manually checking the plot outputs. It's also hidden from users within a sub-function & requires (undocumented) provision of `return.data = TRUE` within the initial `...`.

Reply:
Yes, that's fine. Though don't go overboard with the unit tests, remember you'll be the one needing to change them all then the function changes.

Reply:
These tests are largely already written.  Just with this I can swap the 'expect_pheatmap' + manual check for a direct 'is_equal(output_piece, expected)' or the like.